src/init.c

```diff
@@ -874,6 +874,9 @@ static NOINLINE void _finish_julia_init(JL_IMAGE_SEARCH rel, jl_ptls_t ptls, jl_
         post_boot_hooks();
     }
 
+    if (jl_options.task_timing) {
+        jl_atomic_fetch_add(&jl_task_timing_enabled, 1);
+    }
```

Comment:
wasn't sure where to was best to do this (i put it in `_finish_julia_init` since that's where most of the user-configurable stuff is, and then plumped for "before `jl_start_threads`"), but actually maybe we should do this before `jl_init_root_task`?